packages/account/src/Components/currency-selector/radio-button.tsx

```diff
@@ -1,11 +1,22 @@
-import React from 'react';
+import React, { AllHTMLAttributes, Fragment, InputHTMLAttributes } from 'react';
 import classNames from 'classnames';
 import { Localize } from '@deriv/translations';
 import { Popover, Icon } from '@deriv/components';
 import { getCurrencyDisplayCode } from '@deriv/shared';
+import { TReactElement } from 'Types';
 
-const USTPopover = ({ id }) => {
-    let popover_message;
+type TUSTPopoverProps = {
+    id: string;
+};
+
+interface TRadioButtonProps extends AllHTMLAttributes<HTMLInputElement | HTMLLabelElement> {
```

Comment:
- I suggest to use `type` instead of `interface` to follow what we already have it in the code.
- You can follow the naming convention same as `T{name_of_component}` like `TRadioButton`
- Let's export all the type props into a shared file like common-prop that you've already created.

Reply:
Will make the change